exonum/src/events/tests.rs

```diff
@@ -259,6 +261,33 @@ fn test_network_big_message() {
     assert_eq!(e2.wait_for_disconnect(), first);
 }
 
+#[test]
+fn test_network_max_message_len() {
+    let first = "127.0.0.1:17202".parse().unwrap();
+    let second = "127.0.0.1:17303".parse().unwrap();
+
+    let m1 = raw_message(15, ConsensusConfig::DEFAULT_MESSAGE_MAX_LEN as usize - 1000);
+    let m2 = raw_message(16, ConsensusConfig::DEFAULT_MESSAGE_MAX_LEN as usize + 1000);
```

Comment:
I prefer to declare variables closer to the usage if possible. :smile: Additionally `m1` and `m2` can have more meaningful names. For example, `too_big_message`.

Should we test the message with exactly `DEFAULT_MESSAGE_MAX_LEN` size?